src/main/java/duke/command/ByeCommand.java

```diff
@@ -4,11 +4,21 @@
 import duke.Storage;
 import duke.Ui;
 
+/**
+ * Represents a ByeCommand which exits Duke.
+ */
 public class ByeCommand extends Command {
     public ByeCommand() {
         super(true);
     }
 
+    /**
+     * Executes the ByeCommand and exits Duke.
+     *
+     * @param taskList TaskList.
+     * @param ui Ui.
+     * @param storage Storage.

```

Comment:
Can provide a better description. If a description not needed, then we do not need to include it in the documentation